SingularityService/src/main/java/com/hubspot/singularity/config/SingularityConfiguration.java

```diff
@@ -76,6 +77,8 @@
   @Min(value = 1, message = "Must be positive and non-zero")
   private int defaultBounceExpirationMinutes = 60;
 
+  private String defaultDatePattern = "MMM dd h:mm:ss a zzz";
```

Comment:
If these config properties are going to be used for mail only, they should indicate in the name.